website/content/docs/enterprise/ent-to-ce-downgrades.mdx

```diff
@@ -0,0 +1,144 @@
+---
+layout: docs
+page_title: Downgrade from Consul Enterprise to the community edition
+description: >-
+  Learn how to downgrade your installation of Consul Enterprise to the free Consul community edition (CE)
+---
+
+# Downgrade from Consul Enterprise to the community edition
+
+This document describes how to downgrade from Consul Enterprise to Consul community edition (CE).
+
+## Download the CE binary version
+
+First, download the binary for the CE version you want.
+
+<Tabs>
+<Tab heading="Binary">
+
+All current and past versions of the CE and Enterprise releases are
+available on the [HashiCorp releases page](https://releases.hashicorp.com/consul)
+
+</Tab>
+<Tab heading="Docker">
+
+Docker containers are available on [Docker Hub](https://hub.docker.com/r/hashicorp/consul)
+
+</Tab>
+
+</Tabs>
+
+## Prepare for the downgrade to CE
+
+1. Take a snapshot:
+1. Inspect the snapshot to ensure that Consul successfully captured it using the following command:
+this snapshot somewhere safe. Refer to the following documentation for additional information about creating and using snapshots:
+
+    - [Consul Snapshot](/consul/commands/snapshot)
+    - [Backup Consul Data and State tutorial](/consul/tutorials/production-deploy/backup-and-restore)
+
+1. Temporarily modify your Consul configuration so that its [log_level](/consul/docs/agent/config/cli-flags#_log_level)
+is set to `debug`. After doing this, issue the following command on your servers to
+reload the configuration:
+1. Remove references to services in non-default namespaces and partitions from service resolver, service splitter, and service router configuration entries.
+
+
+## Perform the Downgrade
+
+**1.** Issue the following command to discover which server is currently the leader:
+
+```
+consul operator raft list-peers
+```
+
+You should receive output similar to this (exact formatting and content may differ based on version):
+
+```
+Node       ID                                    Address         State     Voter  RaftProtocol
+dc1-node1  ae15858f-7f5f-4dcb-b7d5-710fdcdd2745  10.11.0.2:8300  leader    true   3
+dc1-node2  20e6be1b-f1cb-4aab-929f-f7d2d43d9a96  10.11.0.3:8300  follower  true   3
+dc1-node3  658c343b-8769-431f-a71a-236f9dbb17b3  10.11.0.4:8300  follower  true   3
+```
+Take note of which agent is the leader.
+
+**2.** Restart or redeploy all clients with a CE version of the binary. You can use a service management system (e.g., systemd, upstart, etc.) to restart the Consul service. If you are not using a service management system, you must restart the  agent manually.
```

Comment:
I think it is better to separate the server and client downgrades. 

For eg: in this section, while the first step is finding the current leader, the second step jumps into downgrading clients which is not related to step 1.